### DISCUSSION TITLE
Make conditional cards work with templates

### DISCUSSION THREAD

**Discussion by b26x** [home-assistant/frontend#11403] [Dashboard]
Adding the option to use an template instead of the state of an entity would improve the option to create an adaptiv UI.
It would replace the necessity to create a template sensor for every card that should hide/show based on a more complex condition.
Even though I'm not an expert I would assume that it could be done without breaking changes.

**Comment by ZephireNZ** [comment]
FYI I have opened https://github.com/home-assistant/frontend/pull/12867 which would resolve this using a template

**Comment by Mariusthvdb** [comment]
yes, please. a much desired feature since long.

**Comment by Mincka** [comment]
I think I remember that this was rejected in the past because it could cause issues in the interface, or maybe it was related to the "entity-filter". I don't remember exactly and I wasn't able to find it again.

The Conditional Card does not get too much love and PR to improve it are often dead ends. :(
https://github.com/home-assistant/frontend/pull/7381
https://github.com/home-assistant/frontend/pull/7688
https://github.com/home-assistant/frontend/pull/10241

I hope your PR will get some love @ZephireNZ, at least some attention.

**Reply by ZephireNZ** [reply]
Yeah given the radio silence on my PR I don't have too much hope...

I can see for #10241 the issue was that attribute updates wouldn't have registered if only an attribute updated but not the state.

Mine makes use of the template renderer, so it would update the same as any template would (which includes when attributes updates).

**Reply by Mincka** [reply]
Indeed, template support would bring the ultimate option for power users. It's also sad for #10241 because they had someone update the PR and do the documentation and in the end, it was just "nope, let's just drop it".
